docs/how_it_works.md

```diff
@@ -61,16 +61,25 @@ in the configuration will be run first.
 E.g. in this case, first the `@expectedException` annotation will be changed to a method,
  then the `setExpectedException` method will be changed to `expectedException`.
 
-```yaml
-# rector.yaml
-services:
-    Rector\PHPUnit\Rector\ExceptionAnnotationRector: ~
+```php
+<?php
+// rector.php
+
+declare(strict_types=1);
 
-    Rector\Renaming\Rector\MethodCall\RenameMethodRector:
-        $oldToNewMethodsByClass:
-            PHPUnit\Framework\TestClass:
-                setExpectedException: 'expectedException'
-                setExpectedExceptionRegExp: 'expectedException'
+use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
+
+return static function (ContainerConfigurator $containerConfigurator): void {
+    $services = $containerConfigurator->services();
+    $services->set(Rector\PHPUnit\Rector\ExceptionAnnotationRector::class);
+    $services->set(Rector\Renaming\Rector\MethodCall\RenameMethodRector::class)
+        ->arg('$oldToNewMethodsByClass', [
+             PHPUnit\Framework\TestClass::class => [
+                'setExpectedException' => 'expectedException',
+                'setExpectedExceptionRegExp' => 'expectedException',
+            ],
+        ]);
+};
```

Comment:
@TomasVotruba Not sure if the services configuration is done correctly here.

Reply:
Looks good to me. What exactly do you mean,

Reply:
I'm curretly refactoring this in #3821 from args to interface method, so it will change this week.

Reply:
Is it the same as it was in yaml configuration before 
```yaml
    Rector\Renaming\Rector\MethodCall\RenameMethodRector:
        $oldToNewMethodsByClass:
            PHPUnit\Framework\TestClass:
                setExpectedException: 'expectedException'
                setExpectedExceptionRegExp: 'expectedException'
```

Reply:
Yes